src/app/dataset-view/dataset.component.ts

```diff
@@ -56,12 +48,12 @@ export class DatasetComponent
     constructor(
         private appDatasetService: DatasetService,
         private router: Router,
-        private activatedRoute: ActivatedRoute,
-        private navigationService: NavigationService,
-        private modalService: ModalService,
+        protected activatedRoute: ActivatedRoute,
```

Comment:
What's the purpose of `protected` modifiers here?

Reply:
In this case, it is best to remove the modifier